packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -541,6 +543,8 @@ class TextStyle extends Diagnosticable {
   /// ![Text height comparison diagram](https://flutter.github.io/assets-for-api-docs/assets/painting/text_height_comparison_diagram.png)
   ///
   /// {@end-tool}
+  ///
```

Comment:
Not specific to this change, but while you're here, can you fix this right above:
![Screen Shot 2019-07-24 at 11 32 21 AM](https://user-images.githubusercontent.com/16964204/61818885-b7e84480-ae06-11e9-9524-87ab443f5bfe.png)
Using a sample snippet wihtout actual code leaves a blank box and other cutoffs.